google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStub.java

```diff
@@ -618,7 +666,9 @@ public EnhancedBigtableStub(EnhancedBigtableStubSettings settings, ClientContext
         callable,
         query,
         settings.bulkReadRowsSettings().getBatchingSettings(),
-        clientContext.getExecutor());
+        clientContext.getExecutor(),
+        null,
+        ctx == null ? clientContext.getDefaultCallContext() : ctx);
```

Comment:
same here,  ctx is never null here

Reply:
Slightly cleaner: Objects.firstNonNull(ctx, clientContext.getDefaultCallContext())